app/models/ont/flowcell.rb

```diff
@@ -12,14 +12,11 @@ class Flowcell < ApplicationRecord
     validates :position,
               presence: true,
               uniqueness: { scope: :ont_run_id,
-                            message: 'should only appear once within run' }
+                            message: I18n.t('errors.messages.uniqueness.flowcell') }
 
-    def self.includes_args(except = nil)
-      args = []
-      args << { library: Ont::Library.includes_args(:flowcell) } unless except == :library
-      args << { run: Ont::Run.includes_args(:flowcells) } unless except == :run
-
-      args
+    # Make table read only. We don't want anything pushing to it.
+    def readonly?
+      true
     end
```

Comment:
I wonder if it would be safer to call `readonly!` in an after_initialize call? That would have the advantage that we also set the `@readonly` instance variable. Doesn't matter too much, as it should all be both temporary and precautionary,.